client/post-editor/edit-post-status/index.jsx

```diff
@@ -54,6 +61,7 @@ var EditPostStatus = React.createClass( {
 		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
 		actions.edit( { sticky: ! this.props.post.sticky } );
+		this.props.toggleStickyStatus( this.props.post.sticky );
```

Comment:
Makes sense, but let's do that cleanup after we get all the actions in place. It would help us evaluate what we have to deal with better.